test/e2e/service.go

```diff
@@ -1099,6 +1099,89 @@ var _ = framework.KubeDescribe("Services", func() {
 			expectedSuccess := nodes.Items[n].Name == readyHostName
 			jig.TestHTTPHealthCheckNodePort(publicIP, healthCheckNodePort, "/healthz", expectedSuccess)
 		}
+
+		By("changing ESIPP annotation off and checking behaviour")
+		// Toggle ESIPP annotation off and check client ip and healthchecks
+		svc = jig.UpdateServiceOrFail(svc.Namespace, svc.Name, func(svc *api.Service) {
+			svc.ObjectMeta.Annotations[service.AnnotationExternalTraffic] =
+				service.AnnotationValueExternalTrafficGlobal
+		})
+		success := false
+		for start := time.Now(); time.Since(start) < kubeProxyLagTimeout; time.Sleep(15 * time.Second) {
+			By("reading clientIP using the TCP service's NodePort")
+			content := jig.GetHTTPContent(nodeIP, tcpNodePort, kubeProxyLagTimeout, "/clientip")
+			clientIP := content.String()
+			framework.Logf("ClientIP detected by target pod using NodePort is %s", clientIP)
+			By("reading clientIP using the TCP service's service port via its external VIP")
+			content = jig.GetHTTPContent(ingressIP, svcTcpPort, kubeProxyLagTimeout, "/clientip")
+			clientIP = content.String()
+			framework.Logf("ClientIP detected by target pod using VIP:SvcPort is %s", clientIP)
```

Comment:
Is there a lower log level that will only show up with some super verbose ginkgo log level ?
That way, whenever I need to debug this test, I can turn on these logs.